File: Bhatkanti/Models/Place_Images.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace Bhatkanti.Models
{
    [Table("place_images")]
    public class Place_Images
    {
        [Key] // Single primary key
        public int Place_Image_Id { get; set; }

        [ForeignKey("Place")]
        public int Place_ID { get; set; }

        [ForeignKey("Images")]
        public int Image_ID { get; set; }

        // Navigation Properties
        public Place Place { get; set; }
        public Images Image { get; set; }
    }
}

File: Backend SpringBoot/src/main/java/com/explorer/Models/PlaceImages.java
package com.explorer.Models;

import jakarta.persistence.*;

@Entity
@Table(name = "place_images")
public class PlaceImages {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int placeImageId;

    @ManyToOne
    @JoinColumn(name = "place_id")
    private Place place;

    @ManyToOne
    @JoinColumn(name = "image_id")
    private Images image;

	
    public PlaceImages(int placeImageId, Place place, Images image) {
		super();
		this.placeImageId = placeImageId;
		this.place = place;
		this.image = image;
	}
    
    public PlaceImages() {
    	
    }


    
	public int getPlaceImageId() {
		return placeImageId;
	}


	public void setPlaceImageId(int placeImageId) {
		this.placeImageId = placeImageId;
	}


	public Place getPlace() {
		return place;
	}


	public void setPlace(Place place) {
		this.place = place;
	}


	public Images getImage() {
		return image;
	}


	public void setImage(Images image) {
		this.image = image;
	}

    
    
    
}

